starport/templates/module/create/oracle/proto/{{moduleName}}/oracle.proto.plush

```diff
@@ -0,0 +1,118 @@
+syntax = "proto3";
```

Comment:
I may think that those types are universal for every potential bandchain modules that would be scaffolded? In this case we could try to ensure those are only generated once.

We can have a separate template in `templates/bandchain` for example and generate when an oracle module is scaffolded for the first time

Reply:
I'm not sure, but I think we can request data from different scripts IDs with other dataset results, let me check it. Because if yes, I think it makes sense to keep the proto buffer there for the user can implement different scripts with other data. What do you think?

Reply:
So would it be something that would be customized by the developer afterward? Because anything that can be customized must actually be specific to the module, so yes we can let it as this

Reply:
yes, we can customize after scaffold to get data from other scripts